can_tools/bootstrap_data/covid_variables.csv

```diff
@@ -200,3 +200,6 @@ total_vaccine_initiated,current,percentage
 total_vaccine_completed,current,percentage
 janssen_vaccine_allocated,cumulative,doses
 janssen_vaccine_completed,new,people
+covid_hospital_admissions,rolling_average_7_day,people_per_100k
+hospital_beds_in_use_covid,rolling_average_7_day,percentage
```

Comment:
I don't care strongly, but do you know if there's a rationale for why we're mixing using covid as a prefix and a suffix?

Reply:
no reason, I didn't mean to break the convention with the above commit--fixing now